### PR TITLE
feature: implement functions that don't need more support from Amplitude-JavaScript except Identify/GroupIdentify

### PR DESCRIPTION
**Pull request by yuhao900914**
implement functions that don't need more support from Amplitude-JavaScript except for Identify/GroupIdentify

### CODE REVIEW

File: lib/amplitude_web.dart [hunk 1]
```diff
@@ -40,6 +71,81 @@ class AmplitudeFlutterPlugin {
...
+          return amplitude.logRevenue(amount, 1, null);
+        }
+      case "identify":
+        {
+          return true;
+        }
+      case "setGroup":
+        {
+          String groupType = args['groupType'];
+          String groupName = args['groupName'];
```
Comment (qingzhuozhen) [inline lib/amplitude_web.dart]: Is our current flutter interface has group name string only? Noticed in JS surface a dynamic. I believe it is fine if Flutter interface only has String accepted.

Comment (qingzhuozhen) [reply lib/amplitude_web.dart]: Take a quick look, seems that is `dynamic`.

**Review by qingzhuozhen — CHANGES_REQUESTED** [review]

File: lib/amplitude_web.dart (hunk 1 above)
Comment (yuhao900914) [reply lib/amplitude_web.dart]: Good find. I change the type to dynamic.

**Review by haoliu-amp — COMMENTED** [review]
The PR title doesn't tell me anything. Could you change it to be more descriptive？

**Review by qingzhuozhen — APPROVED** [review]

**Pull request merged**